setup.py

```diff
@@ -65,11 +133,8 @@
     ],
     python_requires=">=3.6",
     setup_requires=["cython"],
```

Comment:
:+1: I am aware that pyproject.toml is the way to go. I had not yet looked into it yet. I think it is much better as well because you can set a version requirement for setuptools as well. Given that there is some Cython awareness in setuptools as well, that comes in handy.

Reply:
https://github.com/pycompression/python-isal/issues/25

Reply:
Added it. Now I can be 100% certain cython is installed. This makes setup.py slightly simpler.